docs/txp.md

```diff
@@ -64,22 +66,39 @@ tree) and their witnesses.
 In this section we describe parts of GState relevant to transaction
 processing.
 
-* UTXO (unspent transaction outputs). It's a map from `TxIn` to
-  `TxOutAux` which contains all unspent outputs. `TxOutAux` is just an
-  alias for `TxOut`. Later it can be extended if we want to associate
+* UTXO (unspent transaction outputs)
+  ```
+  type Utxo = Map TxIn TxOutAux
+  data TxOutAux = TxOutAux
+     { toaOut   :: !TxOut
+     -- ^ Tx output
+     }
+  ```
+  
+  * It's a map from `TxIn` to
+  `TxOutAux` ([code](https://github.com/input-output-hk/cardano-sl/blob/e7cfb1724024e0db2f25ddd2eb8f8f17c0bc497f/txp/Pos/Txp/Toil/Types.hs#L58)) which contains all unspent outputs.  
+  * `TxOutAux` is just an alias for `TxOut` ([code](https://github.com/input-output-hk/cardano-sl/blob/e7cfb1724024e0db2f25ddd2eb8f8f17c0bc497f/txp/Pos/Txp/Core/Types.hs#L160)). Later it can be extended if we want to associate
   more data with unspent outputs (e. g. slot of the block in which
   this transaction appeared).
-  For example, if a transaction `A` has 1 output (`out1`) which hasn't been
+  * Example, if a transaction `A` has 1 output (`out1`) which hasn't been
   spent yet, UTXO will have a pair `(TxIn (hash A) 0, out1)`.
-* Stakes. They are not needed for tx verification, but they are needed
-  for other parts of the protocol (for example, leader election
-  process called FTS) and they are maintained by tx processing.
-  We store total stake and a map from `StakeholderId` to its stake (`Coin`).
-* Adopted `BlockVersionData`. This data type contains various
-  parameters of the algorithm which can be updated by Update
-  System (details are not covered in this document). Txp doesn't
-  modify this value, it only reads it. Some values are relevant for
-  transaction processing:
+* Stakes
+  ```
+  type StakesMap = HashMap StakeholderId Coin
+  type TotalStake = Coin -- this type doesn't exist in code and put here for convenience of reader
+  ```
+  * Stakes are used by other parts of the protocol (for example, leader election
+  process Follow-the-satoshi or FTS)
```

Comment:
Sentences should end with commas.

Reply:
Should they? It's bullet-list. Default formatting I've seen is capital letters at start and no trailing comma/dot

Reply:
Oh, sorry, I meant dot. It doesn't matter whether it's a bullet-list or not. These are different sentences, they should be separated by dots. If it's all one sentence, then there shouldn't be capital letters in between.

At least for Russian it's true, not sure about English.

Reply:
It turns out English rules are simpler and not having dots is valid 😕